### PR TITLE
FAQ Section

### LINKED ISSUE #128: Adding FAQ Section on Landing Page
### **Is your feature request related to a problem? Please describe.**
I’ve noticed that users often have recurring questions about our services, offerings, and registration process, and they are unsure where to find answers. This leads to frustration and unnecessary support requests, which could be easily avoided with a clear FAQ section.

### **Describe the solution you'd like**
I would like to implement a dedicated **FAQ section** on the landing page to provide users with immediate answers to common questions. This section should be easily accessible, well-organized, and searchable to improve the user experience. It will help users resolve their queries without needing to contact support and will make the overall website more user-friendly.

### **Describe alternatives you've considered**
I have considered the following alternatives:
1. **Pop-up chatbots** that answer common questions, but this might be intrusive to the user experience, especially on a landing page.
2. **Direct links to a support page**, but this increases the number of clicks needed to find answers.
3. **Including FAQs in the footer**, but this might be overlooked by users who expect FAQ sections in the main content.

### **Additional context**
I envision the FAQ section to be divided into categories such as **General Information**, **Registration Process**, **Event Details**, and **Technical Issues**. It will be designed to be responsive and styled to fit the overall aesthetic of the landing page. Below is an image wireframe representing the general layout.

**Timeline:**
- Wireframe submission: **within 1 hour** of approval.
- Design and pull request: **within 24 hours** after the wireframe is approved.

### PR DESCRIPTION
**Pull request by dev129**
This code is for addressing the issue of creation of FAQ section, it addresses the issue number #128

### CODE REVIEW

**Comment by notsoocool** [comment]
You don't fix the font size and also you still have used semibold and bold font style.

File: components/global/header.tsx [hunk 1]
```diff
@@ -108,7 +108,6 @@ export const Header = () => {
             </div>
           </div>
         </div>
-      </div>
```
Comment (notsoocool) [inline components/global/header.tsx:111]: fix this file

**Comment by dev129** [comment]
Okay can we like discuss it together, or do you have an idea in mind ? Like how it should look, so that its easier for me to understand

File: app/images/FAQ Image.png
Comment (notsoocool) [inline app/images/FAQ Image.png:1]: change the picture its blured
used the icon or images from this
https://www.streamlinehq.com/illustrations/new-york-monoline

Comment (dev129) [reply app/images/FAQ Image.png:1]: You are a nice man , I will surely implement this and give a fresh design

**Comment by notsoocool** [comment]
> Okay can we like discuss it together, or do you have an idea in mind ? Like how it should look, so that its easier for me to understand

sure i was give you some examples

**Comment by notsoocool** [comment]
Screenshot 2024-10-22 at 12 42 06 AM
Screenshot 2024-10-22 at 12 42 23 AM
Screenshot 2024-10-22 at 12 42 43 AM

i find these types of design sleek and minimalist with small fonts and less text weight

**Comment by notsoocool** [comment]
Hope you get what i am trying to say, if still you have some confusing we can further discuss it

File: components/global/header.tsx (hunk 1 above)
Comment (dev129) [reply components/global/header.tsx:111]: The header file is now fixed.

File: app/images/FAQ Image.png
Comment (dev129) [reply app/images/FAQ Image.png:1]: I have updated the image and removed the previous one , and that image is responsive to dark and light mode. @notsoocool

**Comment by dev129** [comment]
A major problem the developement server is closing again and again , even upon refreshing or routing to pages. Please resolve this @notsoocool.
See this  

https://github.com/user-attachments/assets/a39507a5-66b1-4a0d-9ef8-8aa84dba440b

**Comment by dev129** [comment]
Hi @notsoocool , the header file div tag is fixed as I have stated about it , the issue here is that the developemetn server is closing again and agin by itself.

**Comment by notsoocool** [comment]
> Hi @notsoocool , the header file div tag is fixed as I have stated about it , the issue here is that the developemetn server is closing again and agin by itself.

can you attach the ss of the starting of the error

**Comment by notsoocool** [comment]
> > Hi @notsoocool , the header file div tag is fixed as I have stated about it , the issue here is that the developemetn server is closing again and agin by itself.

 can you attach the ss of the starting of the error
 because as i can see, when the faq is getting compiled the error comes after that

**Comment by dev129** [comment]
Well thanks alot @notsoocool , for making this bit easier as for a begineer like me its kinda diffcult to understand .

**Comment by notsoocool** [comment]
@dev129 it's working perfectly fine for me. Do one thing: clone it once again, and now run it with **bun** instead of **npm**.

**Comment by dev129** [comment]
Hye @notsoocool , it worked miracously.

**Comment by dev129** [comment]
Hi @notsoocool 

https://github.com/user-attachments/assets/ea9d033b-4201-4bb0-b3d4-4b9a36190fe2

It is responsive to dark and light mode, design is made sleek and minmalist font size is reduced as some points any further reduction means a decline in User Experience, also added a feature so that if some cannot find the desired question then they can still ask for support, hence achieving optimisation.

**Comment by notsoocool** [comment]
@dev129 this looks excellently good, I am impressed.

**Comment by dev129** [comment]
Great now what has to be done ?

**Comment by notsoocool** [comment]
> Great now what has to be done ?

You haven't pushed the changes yet, all done for this issue now, just push the changes what you have made

**Comment by dev129** [comment]
okay

**Comment by dev129** [comment]
Please check now @notsoocool , I hope now there are no further issues.

**Comment by notsoocool** [comment]
Thank you for your contribution! If you'd like to further support the project, feel free to star the repository—it helps the project reach more people and grow.

**Pull request merged**